browser/help/topics/v4-hts.md

```diff
@@ -470,7 +470,7 @@ Row fields
   - `faf`: Filtering allele frequency in the gnomAD genomes.
     - `faf95`: Filtering allele frequency (using Poisson 95% CI).
     - `faf99`: Filtering allele frequency (using Poisson 99% CI).
-  - `grpmax`: Allele frequency information (AC, AN, AF, homozygote count) for the non-bottlenecked genetic ancestry group with maximum allele frequency in the gnomAD genomes. Excludes Ashkenazi Jewish (`asj`), European Finnish (`fin`), Middle Eastern (`mid`), and "Remaining individuals" (`remaining`) groups.
+  - `grpmax`: Allele frequency information (AC, AN, AF, homozygote count) for the non-bottlenecked genetic ancestry group with maximum allele frequency in the gnomAD genomes. Excludes Amish (`ami`), Ashkenazi Jewish (`asj`), European Finnish (`fin`), Middle Eastern (`mid`), and "Remaining individuals" (`remaining`) groups.
```

Comment:
did the genomes not get updated to include mid?

Reply:
I think `mid` is still too small in terms of samples in genomes.